Documentation/project-docs/standard-platform.md

```diff
@@ -292,132 +292,132 @@ Tooling support for the `netstandard` TFM is as follows. This list will be updat
 
 ### Legend 
 - `X` - API appeared in specific .NET Platform Standard version
-- `⇠` - API version determined by nearest `X` e.g. In the table below, if you target .NET Platform Standard version 1.4 and reference Microsoft.CSharp, you'd get the 1.0 API version.
+- `&larr;` - API version determined by nearest `X` e.g. In the table below, if you target .NET Platform Standard version 1.4 and reference Microsoft.CSharp, you'd get the 1.0 API version.
```

Comment:
fwiw, I experienced the unicode arrow problem on one of my boxes too, so this is definitely better :)

I'm not really concerned about the source readability as the markdown isn't formatted for that anyway (look at those tables...).